### PR TITLE
Getting Started with Selenium Automation Testing

### LINKED ISSUE #1161: Getting Started With Selenium Automation Testing
### Brief Summary: 
This article is a continuation of the article ['An Overview on Automation Testing With Selenium'](https://www.section.io/engineering-education/automation-testing-in-selenium/). It aims at taking readers through steps for installing and configuring Selenium web-driver. Readers will get an understanding of how to create and run an automation test in Selenium.

### Key Takeaways: 
1. Introduction
2. Step-by-step guide on how to download and install Selenium web-driver.
3. How to configure Selenium web-driver (with Eclipse)
3. How to create and run the first automation test in Selenium.
4. Conclusion

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)

### PR DESCRIPTION
**Pull request by mbaabuones**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #1161 .

### CODE REVIEW

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by zolomohan** [comment]
@mbaabuones Great article. Please make note of the minor changes I made. Also, Please add Syntax Highlighting to code snippets in the article. 

Also, provide the complete code that you've written in the end.

**Comment by mbaabuones** [comment]
@zolomohan I have finished working on the changes.

File: articles/getting-started-with-selenium-automation-testing/index.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+Let’s test whether a specific URL such as `google.com` functions well. We will conduct this test using the Chrome browser. Every browser used in Selenium testing has a unique browser driver. In our case, we need to download the Chrome driver. This can be downloaded from [here]( https://sites.google.com/a/chromium.org/chromedriver/). Select the Chrome driver that corresponds to your operating system.
+
+![Chrome Driver](/engineering-education/getting-started-with-selenium-automation-testing/chrome-driver.png)
+
+Before launching the browser, we need to set the path of the downloaded executable file. The following is an example of how the property can be set.
+
+```java
+System.setProperty("webdriver.chrome.driver", "D:\\ChromeDriver\\chromedriver.exe");
+```
```
Comment (zolomohan) [inline articles/getting-started-with-selenium-automation-testing/index.md]: @mbaabuones Before starting with this step, please instruct the reader to create a class file and add import statements for necessary classes.

After that, instruct the users to write all of this in the main function.

Comment (mbaabuones) [reply articles/getting-started-with-selenium-automation-testing/index.md]: @zolomohan Done. Thank you for these meaningful insights.

Comment (zolomohan) [reply articles/getting-started-with-selenium-automation-testing/index.md]: It's better to add code snippets every time while explaining. I did it this time, but keep this in mind for your next article.

File: articles/getting-started-with-selenium-automation-testing/index.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    System.setProperty("webdriver.chrome.driver", "D:\\ChromeDriver\\chromedriver.exe");
+
+    //instantiating the driver
+    WebDriver driver = new ChromeDriver();
+
+    //launching the website
+    driver.get("http://www.google.com/"); 
+
+    //running a search
+    driver.findElement(By.name("q")).sendKeys("cheese" + Keys.ENTER);
```
Comment (zolomohan) [inline articles/getting-started-with-selenium-automation-testing/index.md:185]: @mbaabuones You missed adding the import statement for `Keys`.

Comment (mbaabuones) [reply articles/getting-started-with-selenium-automation-testing/index.md:204]: @zolomohan I have added the import statement for `Keys`

**Review by zolomohan — APPROVED** [review]

**Comment by hectorkambow** [comment]
@mbaabuones 👍  great article - it seems like a very detail walk through 🚀 

Please take note of the changes I made to your article for it to flow better (read easier). 

See what edits and suggestions can be applied to future articles moving forward. 

Based on the # of revisions and level of review needed at final - I will be marking this article with a tiered payout label.

Please go over our [resources page](https://github.com/section-io/engineering-education/blob/master/new_contributors/resources-page.md) to see what could help future articles as well. 

We encourage all our students to go over the guides as we want to be sure we are presenting our best work to the community and future potential employers. 

cc @zolomohan 

🚀

**Comment by hectorkambow** [comment]
Also - be sure to include a hero image you would like to use for your articles from a website like pexels.com or unsplash.com 🚀  

(free to use images)

**Pull request merged**